packages/@aws-cdk/aws-ec2/lib/vpn.ts

```diff
@@ -98,6 +130,25 @@ export enum VpnConnectionType {
   DUMMY = 'dummy'
 }
 
+/**
+ * The VPN Gateway that shall be added to the VPC
+ *
+ * @resource AWS::EC2::VPNGateway
+ */
+export class VpnGateway extends cdk.Resource implements IVpnGateway {
+
+  /**
+   * The virtual private gateway Id
+   */
+  readonly gatewayId: string;
+
+  constructor(scope: cdk.Construct, id: string, props: VpnGatewayProps) {
+    super(scope, id);
+
+    const vpnGW = new CfnVPNGateway(this, `${id}`, props);
```

Comment:
The construct ID here should be `"Resource"`.

Reply:
Done. But could you please explain why?

Reply:
It's a rule we've set for L2 constructs. The L1 they wrap should be called "Resource".

During logical ID generation we'll take off that part again.